### DISCUSSION TITLE
Devices are Unhealthy if no Maintenance Window

### DISCUSSION THREAD

**Discussion by mchippy** [SCCM-Zone/CM-SRS-Dashboards#34] [General]
Was just reviewing the reports with a new clients and noticed that if a computer doesn't have a maintenance window, it will never be considered Healthy. Is there a way to optionally exclude that requirement to determine device health? Like putting the DistantMW value to 0 or something?

**Comment by Ioan-Popovici** [comment]
I've thought about adding that for a while now, I'll add the feature, it's and easy fix.

**Comment by Ioan-Popovici** [comment]
![CleanShot 2024-10-25 at 19 59 36@2x](https://github.com/user-attachments/assets/2cd39ba0-353a-419b-9122-73ca05ac4370)

Does this fix it? The first value of the health threshold variable needs to be `0`

**Comment by mchippy** [comment]
Indeed. I didn't try to use the plain text <> and preemptively changed it to the below line and it appears to be working. Thanks! 

`AND @HT_DistantMW &lt;&gt; 0`

![image](https://github.com/user-attachments/assets/e5f9e570-3b1b-45cd-b352-41bbb0db133d)
